starters/basic-starter/pages/index.tsx

```diff
@@ -88,6 +89,23 @@ export default function IndexPage({ menus, events, places }: IndexPageProps) {
 export async function getStaticProps(
   context,
 ): Promise<GetStaticPropsResult<IndexPageProps>> {
+  let index;
+  try {
+    index = await drupal.getIndex();
```

Comment:
I think we should only do this in dev. Do you think that would make sense?

Reply:
What's the reasoning?

Reply:
My thought on this was that folks probably wouldn't want this type of help message to be printed from their production environment since it's not helpful for the end-user.